src/PKSim.Core/Comparison/OntogenyDiffBuilder.cs

```diff
@@ -2,38 +2,40 @@
 using PKSim.Assets;
 using PKSim.Core.Model;
 
-namespace PKSim.Core.Comparison;
-
-public class NullOntogenyDiffBuilder : DiffBuilder<NullOntogeny>
+namespace PKSim.Core.Comparison
 {
```

Comment:
needed to switch from `namespace PKSim.Core.Comparison;` to `namespace PKSim.Core.Comparison {...}` - otherwise it does not compile in my VS2019.

Reply:
yeah that's resharper doing this somehow. Good catch